src/lib/resource/sections/context-mainapplicant-details.js

```diff
@@ -33,15 +33,15 @@ module.exports = {
+                                        '<p class="govuk-body">We\'re going to ask for details about the person who has parental responsibility for the victim. This means they have the legal authority to apply on their behalf.</p><p class="govuk-body">This could be:</p><ul class="govuk-list govuk-list--bullet"><li>the victim\'s birth, step or adoptive parent</li><li>a person named on a Special Guardianship order</li><li>the local authority</li></ul><h2 class="govuk-heading-m">If you do not have parental responsibility</h2><p class="govuk-body">You can still apply even if you do not have parental responsibility for the victim. If this is the case, we will ask you for the details of someone who does have the parental responsibility to act on behalf of the victim.</p><p class="govuk-body">This allows us to communicate with this person or organisation after you’ve sent this application.</p>',
                                     deceased:
-                                        '<p class="govuk-body">We\'re going to ask for details about the person who has parental responsibility for the claimant. This means they have the legal authority to apply on their behalf.</p><p class="govuk-body">This could be:</p><ul class="govuk-list govuk-list--bullet"><li>the claimant\'s birth, step or adoptive parent</li><li>a person named on a Special Guardianship order</li><li>the local authority</li></ul>{{ govukDetails({summaryText: "Can I apply if I do not have parental responsibility?",html: "<p class=\'govuk-body\'>You can still apply even if you do not have parental responsibility for the claimant. This means you have to provide the details of who has authority to act on behalf of the claimant in order to send this application. This allows us to communicate with this person or organisation after you’ve sent this application.</p>"})}}'
+                                        '<p class="govuk-body">We\'re going to ask for details about the person who has parental responsibility for the claimant. This means they have the legal authority to apply on their behalf.</p><p class="govuk-body">This could be:</p><ul class="govuk-list govuk-list--bullet"><li>the claimant\'s birth, step or adoptive parent</li><li>a person named on a Special Guardianship order</li><li>the local authority</li></ul><h2 class="govuk-heading-m">If you do not have parental responsibility</h2><p class="govuk-body">You can still apply even if you do not have parental responsibility for the claimant. If this is the case, we will ask you for the details of someone who does have the parental responsibility to act on behalf of the claimant.</p><p class="govuk-body">This allows us to communicate with this person or organisation after you’ve sent this application.</p>'
```

Comment:
Is it worth fixing this incorrect apostrophe whilst were here?: `you’ve`

Reply:
Apostrophe updated